src/js/components/AlertDialogComponent.jsx

```diff
@@ -39,7 +39,6 @@ var AlertDialogComponent = React.createClass({
 
     return (
       <ModalComponent
-          centered={true}
```

Comment:
All modals will be centered by default, thus there is no need to keep this prop.